edx_proctoring/static/proctoring/js/views/proctored_exam_add_allowance_view.js

```diff
@@ -0,0 +1,170 @@
+var edx = edx || {};
+
+(function (Backbone, $, _, gettext) {
+    'use strict';
+
+    edx.instructor_dashboard = edx.instructor_dashboard || {};
+    edx.instructor_dashboard.proctoring = edx.instructor_dashboard.proctoring || {};
+
+    edx.instructor_dashboard.proctoring.AddAllowanceView = Backbone.ModalView.extend({
+        name: "AddAllowanceView",
+        template: null,
+        template_url: '/static/proctoring/templates/add-new-allowance.underscore',
+        initialize: function (options) {
+            this.proctored_exams = options.proctored_exams;
+            this.proctored_exam_allowance_view = options.proctored_exam_allowance_view;
+            this.course_id = options.course_id;
+            this.model = new edx.instructor_dashboard.proctoring.ProctoredExamAllowanceModel();
+            _.bindAll(this, "render");
+            this.loadTemplateData();
+            //Backbone.Validation.bind( this,  {valid:this.hideError, invalid:this.showError});
+        },
+        events: {
+            "submit form": "addAllowance"
+        },
+        loadTemplateData: function () {
+            var self = this;
+            $.ajax({url: self.template_url, dataType: "html"})
+                .error(function (jqXHR, textStatus, errorThrown) {
+
+                })
+                .done(function (template_data) {
+                    self.template = _.template(template_data);
+                    self.render();
+                    self.showModal();
+                    self.updateCss();
+                });
+        },
+        updateCss: function() {
+            var $el = $(this.el);
+            $el.find('.modal-header').css({
+                "color": "#1580b0",
```

Comment:
It seems odd to have style declarations embedded within the Backbone application -- shouldn't these instead be defined in the SCSS/CSS and the classes swapped?